.docker/entrypoint/lib/commands/verifier.rb

```diff
@@ -3,36 +3,55 @@
 require 'jekyll'
 require 'html-proofer'
 require 'html-proofer-unrendered-markdown'
+require_relative '../file_not_found_error'
 
 # The Jekyll module contains everything related to Jekyll.
 module Jekyll
   # The Jekyll::PlantUml module contains everything related to Jekyll::PlantUml.
   module PlantUml
-    # The Jekyll::PlantUml::Verifier class executes HTMLProofer on a built
-    # Jekyll site in order to verify that the HTML is correct.
-    class Verifier
-      def initialize(jekyll_config)
-        raise 'Nil or empty Jekyll config' if jekyll_config.nil? || jekyll_config.empty?
-        raise "No 'destination' key found in the Jekyll config" unless jekyll_config.key? 'destination'
-
-        @jekyll_destination_dir = jekyll_config['destination']
-        raise "#{@jekyll_destination_dir} does not exist" unless Dir.exist?(@jekyll_destination_dir)
-      end
+    # The Jekyll::PlantUml::Commands module contains the implementations of the
+    # various commands that Jekyll PlantUML supports.
+    module Commands
+      # The Jekyll::PlantUml::Verifier class executes HTMLProofer on a built
+      # Jekyll site in order to verify that the HTML is correct.
+      class Verifier
+        def initialize(jekyll_config)
+          raise ArgumentError, 'jekyll_config cannot be nil' if jekyll_config.nil?
+          raise ArgumentError, 'jekyll_config must be a hash' unless jekyll_config.is_a? Hash
+          raise ArgumentError, 'jekyll_config cannot be empty' if jekyll_config.empty?
+          raise ArgumentError, "No 'destination' key found in the Jekyll config" unless jekyll_config.key? 'destination'
+
+          @jekyll_destination_dir = jekyll_config['destination']
+          unless Dir.exist?(@jekyll_destination_dir)
```

Comment:
Codacy found an issue: [Use a guard clause (`return if Dir.exist?(@jekyll_destination_dir)`) instead of wrapping the code inside a conditional expression.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)